pom.xml

```diff
@@ -440,11 +440,6 @@
             <artifactId>slf4j-log4j12</artifactId>
             <version>1.6.1</version>
         </dependency -->
-        <dependency>
```

Comment:
FWIW: This was added in #2964 but I don't see any actual use of it in that PR. Haven't checked the other pom additions from it, but they aren't dependabotted either.